sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/DirectConnectionConfig.java

```diff
@@ -98,49 +117,57 @@ public DirectConnectionConfig setIdleEndpointTimeout(Duration idleEndpointTimeou
     }
 
     /**
-     * Gets the max channels per endpoint
-     * @return the max channels per endpoint
+     * Gets the max connections per endpoint
+     * Default value is 30
+     *
+     * @return the max connections per endpoint
      */
-    public int getMaxChannelsPerEndpoint() {
-        return maxChannelsPerEndpoint;
+    public int getMaxConnectionsPerEndpoint() {
+        return maxConnectionsPerEndpoint;
     }
 
     /**
-     * Sets the max channels per endpoint
-     * @param maxChannelsPerEndpoint the max channels per endpoint
+     * Sets the max connections per endpoint
+     * Default value is 30
+     *
+     * @param maxConnectionsPerEndpoint the max connections per endpoint
      * @return the {@link DirectConnectionConfig}
      */
-    public DirectConnectionConfig setMaxChannelsPerEndpoint(int maxChannelsPerEndpoint) {
-        this.maxChannelsPerEndpoint = maxChannelsPerEndpoint;
+    public DirectConnectionConfig setMaxConnectionsPerEndpoint(int maxConnectionsPerEndpoint) {
+        this.maxConnectionsPerEndpoint = maxConnectionsPerEndpoint;
         return this;
     }
 
     /**
-     * Gets the max requests per endpoint
+     * Gets the max requests per connection per endpoint
+     * Default value is 10
+     *
      * @return the max requests per endpoint
      */
-    public int getMaxRequestsPerChannel() {
-        return maxRequestsPerChannel;
+    public int getMaxRequestsPerConnection() {
```

Comment:
if the user switches to DirectHttps some of these will not be relevant.

Reply:
Yups, `DirectHttps` is anyway not supported programatically. We also don't want to encourage it, but I get your point. 
May be we should call this config as `DirectTcpConnectionConfig` ?

Reply:
Direct from CX will always be TCP. 
Https might be hacky workaround which is best avoided (Service team doesn't recommended it). 

Refering HTTPS in code docs if-any will lead a ambiguity/confuson. 

If Https is configured and some are not applicable, one options to fail at runtime which is okey.